db/schema_tables.cc

```diff
@@ -305,23 +305,31 @@ schema_ptr tables() {
 
 // Holds Scylla-specific table metadata.
```

Comment:
The changelog shoud explain about what this code does. I happen to know it will, but I'm not the only one who will try to read this commit one day and wonder about it.

Reply:
Looks like the changelog thinks the column-censoring stuff is trivial.